compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaTupleMemberSymbol.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.ballerina.compiler.api.impl.symbols;
+
+import io.ballerina.compiler.api.impl.SymbolFactory;
+import io.ballerina.compiler.api.symbols.AnnotationAttachmentSymbol;
+import io.ballerina.compiler.api.symbols.AnnotationSymbol;
+import io.ballerina.compiler.api.symbols.SymbolKind;
+import io.ballerina.compiler.api.symbols.TupleMemberSymbol;
+import io.ballerina.compiler.api.symbols.TypeSymbol;
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BAnnotationAttachmentSymbol;
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BVarSymbol;
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Represents a tuple member symbol.
+ *
+ * @since 2201.4.0
+ */
+public class BallerinaTupleMemberSymbol extends BallerinaSymbol implements TupleMemberSymbol {
+
+    private List<AnnotationSymbol> annots;
+    private List<AnnotationAttachmentSymbol> annotAttachments;
+    private final TypeSymbol type;
+
+    public BallerinaTupleMemberSymbol(CompilerContext context,
+                                      String name,
+                                      BVarSymbol symbol,
+                                      TypeSymbol type) {
+        super(name, SymbolKind.TUPLE_MEMBER, symbol, context);
```

Comment:
Do we have other symbols where the name is empty? If it is empty, will there be an impact on tooling? Asking because the name doesn't really make sense for a tuple member, and exposing it won't be ideal.

Reply:
The semantic API symbols are modeled to always associate with a name.  One thing we can do here is, we can expose an empty string via `getName()` API

Reply:
So just to clarify, internally we have a name, but `getName()` returns empty string? Would that still work for tooling?

Reply:
Yeah. That's okay

Reply:
+1 to do that then. Do we have tests for a scenario or two (that depends on name)?

Reply:
For tuple members, we don't need, right?

Reply:
For other symbols we do have tests for name

Reply:
How do we test 

> The semantic API symbols are modeled to always associate with a name. 

then?

Reply:
Had an offline chat with @MaryamZi regarding this. We decided to pass `null` or empty string when creating the tuple-member symbol